### PR TITLE
Update rulePriority for Azure

### PR DESCRIPTION
**Pull request by Anandkumar26**
- Nephe created security rules will have priority starting from 1000

### CODE REVIEW

File: pkg/cloud-provider/cloudapi/azure/azure_security.go [hunk 1]
```diff
@@ -297,9 +297,14 @@ func (computeCfg *computeServiceConfig) buildEffectiveNSGSecurityRulesToApply(ap
 			continue
 		}
 		// skip any rules not created by nephe.
```
Comment (reachjainrahul) [inline pkg/cloud-provider/cloudapi/azure/azure_security.go:299]: nephe rules will be created from ruleStartPriority.

File: pkg/cloud-provider/cloudapi/azure/azure_security.go [hunk 2]
```diff
@@ -297,9 +297,14 @@ func (computeCfg *computeServiceConfig) buildEffectiveNSGSecurityRulesToApply(ap
...
+		if *rule.Properties.Priority < ruleStartPriority {
+			continue
+		}
+		// skip any rules not created by nephe.
```
Comment (reachjainrahul) [inline pkg/cloud-provider/cloudapi/azure/azure_security.go:303]: combine both in 1 line with || ?

File: pkg/cloud-provider/cloudapi/azure/azure_security.go (hunk 1 above)
Comment (Anandkumar26) [reply pkg/cloud-provider/cloudapi/azure/azure_security.go]: Done

File: pkg/cloud-provider/cloudapi/azure/azure_security.go (hunk 2 above)
Comment (Anandkumar26) [reply pkg/cloud-provider/cloudapi/azure/azure_security.go]: Done

**Review by reachjainrahul — APPROVED** [review]
/LGTM

**Review by reachjainrahul — APPROVED** [review]
/LGTM

**Pull request merged**